src/Data.cpp

```diff
@@ -503,11 +504,8 @@ int Data::loadWindowBin()
 bool Data::load()
 {
 	if (char_names.isEmpty()) {
-		char_names
-				<< QObject::tr("Cloud") << QObject::tr("Barret") << QObject::tr("Tifa")
-				<< QObject::tr("Aerith") << QObject::tr("Red XIII") << QObject::tr("Yuffie")
-				<< QObject::tr("Cait Sith") << QObject::tr("Vincent") << QObject::tr("Cid")
-				<< QObject::tr("Yound Cloud") << QObject::tr("Sephiroth") << QObject::tr("Chocobo");
+        for(int i=0; i< 12;i++)

```

Comment:
Please use the new syntax conventions of this project: for (int i = 0; i < 12; ++i) with brackets everytime